src/api/apiConstants.ts

```diff
@@ -0,0 +1,5 @@
+const BUSINESS_URL = '/v3/businesses';
+
+const SEARCH_BUSINESS = BUSINESS_URL + '/search';
+
+export {SEARCH_BUSINESS};
```

Comment:
Do not export individual const like this. 

Create const ApiUrlEndPoints = { business: { search: "/v3/businesses/search" }}

ApiUrlEndPoints.business.search